pygmt/src/grd2xyz.py

```diff
@@ -126,6 +126,18 @@ def grd2xyz(grid, output_type="pandas", outfile=None, **kwargs):
         - :class:`pandas.DataFrame` or :class:`numpy.ndarray` if ``outfile`` is
           not set (depends on ``output_type``)
 
+    Examples
+    --------
+    >>> import pygmt  # doctest: +SKIP
+    >>> # Load a grid of @earth_relief_30m data, with an x-range of 10 to 30,
+    >>> # and a y-range of 15 to 25
+    >>> grid = pygmt.datasets.load_earth_relief(
+    ...     resolution="30m", region=[10, 30, 15, 25]
+    ... )  # doctest: +SKIP
+    >>> # Create a pandas DataFrame with the xyz data from an input grid
+    >>> xyz_array = pygmt.grd2xyz(
+    ...     grid=grid, output_type="pandas"
+    ... )  # doctest: +SKIP
     """
```

Comment:
Maybe print two rows of the dataframe so people can see what happened.

```suggestion
    >>> print(xyz_dataframe.head(n=2))  # doctest: +SKIP
         lon    lat  elevation
    0  10.25  24.75      903.5
    1  10.75  24.75      820.0
    """
```

Reply:
Why use a print statement here rather than just `xyz_dataframe.head(n=2)`? The doctest would pass with or without it and I think most of our examples assume the use of a jupyter notebook such that the visible output for those copying the example would be the same as well.

Reply:
Or we can simply use doctest's `ELLIPSIS` directive:
```
>>> xyz_dataframe  # doctest: +ELLIPSIS
         lon    lat  elevation
    0  10.25  24.75      903.5
    1  10.75  24.75      820.0
    ...
```

I don't test it, so not 100% sure it works.

Reply:
Yes you're right, we can omit the `print()`. I was just running this in Jupyter and wanted the text output instead of the fancy html output.

Reply:
Should I leave this branch as is?